uber/static/theme/prereg.css

```diff
@@ -15,10 +15,6 @@ body {
     background-repeat: repeat-x repeat-y;
 }
 
```

Comment:
Because of the new form rows, we've adjusted our padding rules slightly.  Things pretty much look the same now as they did before.